prime-router/settings/prod/0163-nv-prod-up-onboarding.yml

```diff
@@ -0,0 +1,109 @@
+- name: "nv-phd"
+  description: "Nevada Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "NV"
+  countyName: null
+  filters: []
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:    
+  - name: "full-elr"
+    organizationName: "nv-phd"
+    topic: "full-elr"
+    customerStatus: "inactive"
+    translation: !<HL7>
+      schemaName: "metadata/hl7_mapping/receivers/STLTs/NV/NV-receiver-transform"
+      useTestProcessingMode: false
+      useBatchHeaders: true
+      receivingApplicationName: "NVDOH"
+      receivingApplicationOID: null
+      receivingFacilityName: "NVODH"
```

Comment:
Doesn't this also go into the transform? And is it supposed to be ODH?

Reply:
Yes, you are correct as above.